### ISSUE TITLE
GitHub Action/Workflow to build the site

### ISSUE THREAD

**Issue by ns-rse** [RSE-Sheffield/collaborative_github_exercise#495]
Its tedious and prone to not working having to build the site locally, commit the resultant HTML output and manually upload it.

Would be better if there is a GitHub Action that can be manually triggered to build and deploy the update.

**Comment by ubdbra001** [comment]
Something like [this](https://github.com/simonrolph/render-rmarkdown-with-actions/blob/main/.github/workflows/main.yml)?

**Comment by ns-rse** [comment]
Yep, that could work as a basis for what we need, would need a few tweaks but a good starting point.

<aside>
Simon gave a talk at SheffieldR group meetings a couple of years ago on the `target` package.
</aside>

**Comment by ns-rse** [comment]
Closed by #569
